src/sonic-config-engine/minigraph.py

```diff
@@ -728,8 +728,12 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['DHCP_SERVER'] = dict((item, {}) for item in dhcp_servers)
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
-
     results['ACL_TABLE'] = acls
+    results['FEATURES'] = {
+        'telemetry': {
+            'status': 'on'
```

Comment:
I think the values should be "enabled"/"disabled" instead of "on"/"off"

Reply:
It's not acceptable to enable this service by default.

Reply:
changed value to enabled as discussed